.github/workflows/main.yml

```
@@ -0,0 +1,40 @@
+name: Lighthouse CI
+on: pull_request
+jobs:
+  lighthouse:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Wait for build to succeed
+        uses: fountainhead/action-wait-for-check@v1.0.0
+        id: wait-for-build
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          checkName: Build
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+          intervalSeconds: 120
+
+      - name: Failed build
+        if: steps.wait-for-build.outputs.conclusion == 'failure' || steps.wait-for-build.outputs.conclusion == 'timed_out'
+        run: exit 1
+
+      - uses: actions/checkout@v2
+
+      - name: Use node 14
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+          cache: 'yarn'
+
+      - run: yarn install
+
+      - name: Setup BASE_SITE_URL Env Var.
+        run: |
+          echo "BASE_SITE_URL=https://sfj-$(git rev-parse --short ${{ github.event.pull_request.head.sha }})--base.preview.vtex.app" >> $GITHUB_ENV
```

I wonder if this will fix [this](https://github.com/vtex-sites/storeframework.store/runs/5816642803?check_suite_focus=true#step:8:30) LH GH Action failure:

```suggestion
          echo "BASE_SITE_URL=https://sfj-$(git rev-parse --short ${{ github.event.pull_request.head.sha }})--storeframework.preview.vtex.app" >> $GITHUB_ENV
```

---

I was looking for a smart way to use [this](https://docs.github.com/en/actions/learn-github-actions/contexts#:~:text=for%20GitHub%20Actions.%22-,github.repository,-string) `github.repository` to avoid hardcoding `base`/`storeframework` but couldn't find much about how to split the string (I think it would return `"vtex-sites/storeframework.store"` and we need just `"storeframework"`).

---

I've fixed it as you suggest on #39. I think we can improve how it's handled in a later moment